### PR TITLE
Update RetinaNet to use better LR schedule and PredictionDecoder configuration

### PR DESCRIPTION
**Pull request by LukeWood**

### CODE REVIEW

File: guides/md/keras_cv/retina_net_overview.md [hunk 1]
```diff
@@ -124,57 +118,6 @@ train_ds = train_ds.map(unpackage_tfds_inputs, num_parallel_calls=tf.data.AUTOTU
 eval_ds = eval_ds.map(unpackage_tfds_inputs, num_parallel_calls=tf.data.AUTOTUNE)
 ```
 
-<div class="k-default-codeblock">
-```
-[1mDownloading and preparing dataset 868.85 MiB (download: 868.85 MiB, generated: Unknown size, total: 868.85 MiB) to ~/tensorflow_datasets/voc/2007/4.0.0...[0m
```
Comment (fchollet) [inline guides/md/keras_cv/retina_net_overview.md:129]: Please manually remove unnecessary logs (it harm readability)

Comment (LukeWood) [reply guides/md/keras_cv/retina_net_overview.md:129]: Done

**Comment by LukeWood** [comment]
@fchollet should be ready

File: guides/md/keras_cv/retina_net_overview.md (hunk 1 above)
Comment (fchollet) [reply guides/md/keras_cv/retina_net_overview.md:129]: But not pushed?

Comment (LukeWood) [reply guides/md/keras_cv/retina_net_overview.md:129]: this is a removal not an addition, right?

Comment (LukeWood) [reply guides/md/keras_cv/retina_net_overview.md:129]: it was in the guide last render - but I removed in this PR

**Review by fchollet — APPROVED** [review]
LGTM, thanks

**Pull request merged**